kore/src/Kore/Reachability/Claim.hs

```diff
@@ -339,7 +337,10 @@ transitionRule stuckCheck claims axiomGroups = transitionRuleWorker
     transitionRuleWorker Begin claimState = pure claimState
     transitionRuleWorker Simplify claimState
         | Just claim <- retractSimplifiable claimState =
-            Transition.ifte (simplify claim) (pure . ($>) claimState) (pure Proven)
+            Transition.ifte
+                (simplify claim)
+                (pure . ($>) claimState)
+                (trace ("\nDebug: LHS simplified to Bottom:\n") $ pure Proven) -- <> (show . pretty) (from @_ @ClaimPattern claim)) $ pure Proven)
```

Comment:
From @jberthold :

Try `traceM blabla >> pure Proven` because that argument will get evaluated anyway.

Or `pure (trace blabla Proven)`

The trace message will get printed when this computation is ran and the caller checks the result.

Reply:
@goodlyrottenapple : pure (....) won't work because it's being floated out